graylog2-web-interface/src/stores/PaginationTypes.js

```diff
@@ -31,6 +31,13 @@ export type ListPagination = Pagination & {
   count: number,
 };
 
+export const DEFAULT_PAGINATION = {
+  page: 1,
+  perPage: 10,
+  query: '',
+  total: 0,
+};
```

Comment:
Do we need `total` here? The lists we've implemented recently, receive the total amount from the request response.
To ensure the `DEFAULT_PAGINATION` has the correct format, we could type it as `Pagination`.